projects/m1/002-bottle-deposits/solutions/GiorgiaSandionigi/index.js

```diff
@@ -0,0 +1,19 @@
+// definisco i contenitori
+
+const depositSmallContainer = 0,10;
+const depositBigContainer = 0,25;
```

Comment:
In javascript floating numbers are represented with a dot instead of a comma, the business logic here is ok but you need to change `0,10` => `0.10` and `0,25` => `0.25` if you don't want any syntax error